packages/muelu/research/regionMG/examples/structured/Driver_Structured_Regions.cpp

```diff
@@ -795,7 +792,7 @@ int main_(Teuchos::CommandLineProcessor &clp, Xpetra::UnderlyingLib& lib, int ar
     }
 
     // Richardson iterations
-    typename Teuchos::ScalarTraits<Scalar>::magnitudeType normResIni;
+    typename Teuchos::ScalarTraits<Scalar>::magnitudeType normResIni = Teuchos::ScalarTraits<Scalar>::zero();
```

Comment:
Initializing with empty curly braces has the same effect:
```c++
typename Teuchos::ScalarTraits<Scalar>::magnitudeType normResIni {};
```
I've started doing this more as a way to reduce dependencies on headers external to my package or application.

Reply:
I'll keep this in mind. Thanks!